File: Selenium/Project/Activity3.java
package Projects_HRM;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.junit.AfterClass;
import org.junit.Assert;
import org.junit.BeforeClass;
import org.junit.Test;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity3 {
    static WebDriver driver;

    @BeforeClass
    public static void setup() {
        // Set up the Firefox driver
        WebDriverManager.firefoxdriver().setup();
        //Create a new instance of the Firefox driver
        driver = new FirefoxDriver();
        driver.get("http://alchemy.hguy.co/orangehrm");
    }

    @Test
    public void verifyHomepage() {
        WebElement username = driver.findElement(By.xpath("//input[@id = 'txtUsername']"));
        WebElement password = driver.findElement(By.xpath("//input[@id = 'txtPassword']"));
        username.sendKeys("orange");
        password.sendKeys("orangepassword123");
        driver.findElement(By.xpath("//input[@id = 'btnLogin']")).click();
        WebElement welcome = driver.findElement(By.xpath("//a[@id = 'welcome']"));
        Assert.assertEquals(welcome.getText(), "Welcome Abhi");
    }

    @AfterClass
    public static void afterMethod() {
        //Close the browser
        driver.quit();
    }

}

File: Selenium/Project/Activity3.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

# Set up the Firefox Driver with WebDriverManger
service = FirefoxService(GeckoDriverManager().install())
with webdriver.Firefox(service=service) as driver:
    driver.get("http://alchemy.hguy.co/orangehrm")
    username = driver.find_element(By.XPATH, "//input[@id = 'txtUsername']")
    password = driver.find_element(By.XPATH, "//input[@id = 'txtPassword']")
    username.send_keys("orange")
    password.send_keys("orangepassword123")
    driver.find_element(By.XPATH, "//input[@id = 'btnLogin']").click()
    welcome = driver.find_element(By.XPATH, "//a[@id = 'welcome']")
    assert welcome.text == "Welcome Abhi"

    driver.quit()